.gitignore

```diff
@@ -12,7 +12,6 @@
 /build
 /dist
 /package
-/scheme/index.js
```

Comment:
i thought we were going to stop pulling in this file

Reply:
Nope, it turns out the dataset is made from two even bigger sets that would have to be included to get the post install script to work, and this file is in the package when you get it from npm